collections/nemo_nlp/nemo_nlp/text_data_utils.py

```diff
@@ -619,17 +623,104 @@ def process_wkt(data_dir):
     return data_dir
 
 
+def download_wkt2(data_dir):
+    os.makedirs('data/lm', exist_ok=True)
+    logger.warning(f'Data not found at {data_dir}. '
+                   f'Download {dataset_name} to data/lm')
+    data_dir = 'data/lm/wikitext-2'
+    subprocess.call('scripts/get_wkt2.sh')
+    return data_dir
+
+
 class LanguageModelDataDesc:
     def __init__(self, dataset_name, data_dir, do_lower_case):
         if dataset_name == 'wikitext-2':
             if not os.path.exists(data_dir):
-                os.makedirs('data/lm', exist_ok=True)
-                logger.warning(f'Data not found at {data_dir}. '
```

Comment:
Let's keep this message but make a logger.info

Reply:
I think data not found deserves a warning